tf_files/aws/variables.tf

```diff
@@ -37,9 +42,22 @@ variable "kube_ssh_key" {
    kubernete nodes, Example:
    '- ssh-rsa XXXX\n - ssh-rsa XXX' */
 variable "kube_additional_keys" {
-    default = ""
+    default = <<EOB
```

Comment:
let's not add those keys as default? admins don't need to ssh onto pods

Reply:
ok